src/lib.rs

```diff
@@ -495,6 +494,10 @@ mod macros;
 #[cfg(feature = "experimental-inspect")]
 pub mod inspect;
 
+// Putting the declaration of prelude at the end seems to help encourage rustdoc to prefer using
+// other paths to the same items. (e.g. `pyo3::types::PyAnyMethods` instead of `pyo3::prelude::PyAnyMethods`).
+pub mod prelude;
```

Comment:
Ah! Looks like moving this here will require updating some of the UI tests.

Reply:
Ah good spot; the UI test change also seems like an improvement to me, so that's nice!